src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -0,0 +1,197 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+
+object BlockVTK {
+
+  private val EPSILON = 1.0e-6
+
+  /**
+   * Calculates the rotation matrix to rotate the input plane (specified by its normal)
```

Comment:
Cool, sorry about that. Intellij was inserting those automatically, so just assumed that's what we were doing. Will fix this to be consistent.